fe/fe-core/src/main/java/org/apache/doris/common/profile/ProfileTreeBuilder.java

```diff
@@ -242,7 +246,7 @@ private ProfileTreeNode buildSingleInstanceTree(RuntimeProfile instanceProfile,
                     || profile.getName().startsWith(PROFILE_NAME_VDATA_STREAM_SENDER)
                     || profile.getName().startsWith(PROFILE_NAME_VDATA_BUFFER_SENDER)
                     || profile.getName().startsWith(PROFILE_NAME_DATA_BUFFER_SENDER)
-                    || profile.getName().startsWith(PROFILE_NAME_OLAP_TABLE_SINK)) {
+                    || profile.getName().startsWith(PROFILE_NAME_VOLAP_TABLE_SINK)) {
```

Comment:
`VOlapTableSinkV2` also starts with `VOlapTableSink`, but I guess it's expected to share this logic?

Reply:
so ..... rubbish logic...
I will add this case